src/main/webapp/index.js

```diff
@@ -0,0 +1,82 @@
+const firebaseApp = firebase.initializeApp({
+    apiKey: "AIzaSyAv-Zy1ZpAwC31yVhtcSgGIj3OmTUbkE5Y",
```

Comment:
It's not recommended to write apiKeys directly into the code (security reasons), you can research a little about how to set it as an environment variable, I'll allow it for now since this is just a test project but if you have time please change it to an environment variable and get a new key (because this is committed it will always be a vulnerability if we keep using it)

Reply:
I see, I'll definitely look into this.